youtube-dl.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Paylists
+# 2019: https://www.youtube.com/playlist?list=PLhEVvej0BBEI3YMccR3_CLDiBSNvGFv5I
+# 2002: https://www.youtube.com/playlist?list=PLEVVO3zRKfv_XK06fOi41FQVATxgOHs6o
+
+year=2020
+playlist="https://www.youtube.com/playlist?list=PLEVVO3zRKfv_XK06fOi41FQVATxgOHs6o"
+source_directory="2020Lockout"
+dest_directory="2020"
+
+### Santatize Data Input ###
+echo -n "Before starting, copy list from Evernote to evernote-list.txt"
+pause
+
+echo -n "Removing all lines that start with #'s"
+# Remove all lines that start with #
+while read input; do
+  sed '/^#.*/d' "$input"
+done <evernote-list.txt
+cp evernote-list.txt final-list.txt
+
+# replace all " - " with ","'s'
+while read input; do
+  sed '/^#.*/d' "$input"
```

Comment:
Update regexp for `-` vs `#`

Reply:
Also, one of my favorite tools is 
https://regex101.com/ and their test cases feature. It's saved me quite a few times.